services/github/github-contributors.service.js

```diff
@@ -8,6 +8,11 @@ import { documentation, httpErrorsFor } from './github-helpers.js'
 // All we do is check its length.
 const schema = Joi.array().items(Joi.object())
 
+const descriptionWithNote = `${documentation}
+
+Note: Co-authors are not included in the count due to endpoint limitations.
+`
```

Comment:
How about using `.join('\n\n')` here for a better format in the code base?

```js
const description = [
	documentation,
	'Note: Co-authors are not included in the count due to endpoint limitations.',
].join('\n\n')
```

Reply:
It looks more elegant, but I think it might be less obvious fir some.
I don't have a strong opinion but I think i will adopt it here.
Regarding performance I think this has negligible effect on compile time. So I doubt this should be a consideration.